Sources/scipio/CreateCommands.swift

```diff
@@ -52,12 +52,4 @@ extension Scipio {
 
 private let availablePlatforms: Set<SDK> = [.iOS, .macOS, .tvOS, .watchOS]
 
-#if compiler(>=6.0)
-
-extension Runner.Options.Platform: @retroactive ExpressibleByArgument { }
-
-#else
-
 extension Runner.Options.Platform: ExpressibleByArgument { }
-
-#endif
```

Comment:
<img width="1106" alt="Screenshot 2024-09-09 at 17 04 54" src="https://github.com/user-attachments/assets/b2149c28-711f-4835-b0b5-3caddf7d9ad5">

These changes are not needed anymore since `Beta 6`.

Reply:
These incompatibilities are already resolved on #141. Please rebase the PR.